.circleci/src/jobs/build_martin.yml

```diff
@@ -2,6 +2,9 @@ docker:
   - image: rust:bullseye
 working_directory: ~/martin
 steps:
+  - checkout:
+      path: ~/project
+  - run: apt update -y && apt install -y nodejs npm
```

Comment:
🔧  i think it would be nice to add a name to the run step to clarify what will be done like "Install prerequisite packages" or sth similar